### ISSUE TITLE
Add !kick command to start a callvote

### ISSUE THREAD

**Issue by atomy** [algo7/tf2_rcon_misc#25]
callvote kick <name>

Autocomplete name, if multiple names possible maybe print error to chat.
Check if command itself already autocompletes.

**Comment by atomy** [comment]
Console command:
> callvote kick (name)

**Comment by atomy** [comment]
<html><body>
<h3><span class="mw-headline" id="callvote">callvote</span></h3>
<p><code class="qmsg"><b>Note:</b> This command can take 0 arguments.  If it does, a VoteSetup usermessage is returned to that user.</code><br>
</p>


Name: | callvote
-- | --
Structure: | string  type  "Kick", "RestartGame", "ChangeLevel", "NextLevel",  "ScrambleTeams", or "ChangeMission".  Should return VoteSetup  Usermessage if left blank.   string  param1  "userid_short kicktype_string" for Kick, map name for Changelevel/NextLevel, or popfile for ChangeMission. Otherwise, left blank | string | type | "Kick", "RestartGame", "ChangeLevel", "NextLevel",  "ScrambleTeams", or "ChangeMission".  Should return VoteSetup  Usermessage if left blank. | string | param1 | "userid_short kicktype_string" for Kick, map name for Changelevel/NextLevel, or popfile for ChangeMission. Otherwise, left blank
string | type | "Kick", "RestartGame", "ChangeLevel", "NextLevel",  "ScrambleTeams", or "ChangeMission".  Should return VoteSetup  Usermessage if left blank.
string | param1 | "userid_short kicktype_string" for Kick, map name for Changelevel/NextLevel, or popfile for ChangeMission. Otherwise, left blank


<p><br>
</p><p>kicktype_string is one of these (at least in the English version):
</p>
<ul><li>other</li>
<li>cheating</li>
<li>idle</li>
<li>scamming</li></ul>
</body>
</html>

https://wiki.alliedmods.net/Tf2_voting

**Comment by algo7** [comment]
Autocomplete when typing in chat?

**Comment by atomy** [comment]
Sort of implicit autocomplete, like !kick at and it kicks atomy cause it's the only player matching.

**Comment by algo7** [comment]
We can use something like https://en.m.wikipedia.org/wiki/Levenshtein_distance 

or other similar implementation. Some go NLP libs must have some edit distance algos included. Otherwise we have to roll our own. Possible edge cases (probably not so edge case) are players with names compose of non-latin characters.

**Comment by atomy** [comment]
Yeah maybe, was thinking of a simple strings.Contains() looping through all current players and if multiple matches it needs to error and not do anything. Or even hide error cause there's no real place to show error w/o annoying everyone else.

reason being optional

!kick ato cheat
translates to
callvote kick "atomy" cheating

!kick ato
translates to
callvote kick "atomy" other
